server/src/graphql/mutations/createApp.ts

```diff
@@ -1,19 +1,9 @@
-import * as yup from 'yup';
 import { MutationResolvers } from '../../generated/graphql';
 import { prisma } from '../../prisma';
 import { dokku } from '../../lib/dokku';
 import { buildAppQueue } from '../../queues/buildApp';
 import { sshConnect } from '../../lib/ssh';
-
-// Validate the name to make sure there are no security risks by adding it to the ssh exec command.
-// Only letters and "-" allowed
-// TODO unit test this schema
-const createAppSchema = yup.object({
```

Comment:
@Akirtovskis the schema will really soon be different, it's a good idea to share the name validation between the mutations but maybe we can do something like:
```
const createAppSchema = yup.object({
 name: importedFromFile,
});
```
What do you think?

Reply:
@pradel what does `importedFromFile` mean in this case? Is it a separate yup schema that we will be exporting from somewhere?

Reply:
Yup eg:
```
# utils.ts
const importedFromFile =  yup
    .string()
    .required()
    .matches(/^[a-z0-9-]+$/)

# createApp.ts
const createAppSchema = yup.object({
 name: importedFromFile,
});
```

Reply:
Sounds great, will do.

Reply:
Done.